### PR TITLE
web: Visually improve the "Add file system" menu

### PR DESCRIPTION
**Pull request by dgdavid**
During the today's review, we found a little thing to improve

* The visual presentation of the _Add file system_ menu options.

  | Before | After |
  |-|-|
  | Screenshot from 2024-04-29 11-02-39 | Screenshot from 2024-04-29 11-02-31

## Note for reviewers

Please, feel free to jump into the PR and change whatever you think can be improved.

### CODE REVIEW

File: web/src/components/storage/VolumeDialog.jsx [hunk 1]
```diff
@@ -720,7 +720,7 @@ export default function VolumeDialog({
 
   return (
     /** @fixme blockSize medium is too big and small is too small. */
-    <Popup title={title} isOpen={isOpen} blockSize="medium" inlineSize="medium">
+    <Popup title={title} isOpen={isOpen} blockSize="large" inlineSize="medium">
```
Comment (ancorgs) [inline web/src/components/storage/VolumeDialog.jsx:723]: The comment above saying "medium is too big" it's clearly outdated now.

**Comment by ancorgs** [comment]
| Before | After |
  |-|-|
  | before_swap | after_swap |
  | before_home | after_home |

**Comment by dgdavid** [comment]
BTW, when present, auto should be the first size option.

**Comment by dgdavid** [comment]
Or even using columns for size only, 

Screenshot from 2024-04-30 09-07-17


There are many options for fixing the issue when rethinking the poor design of the form.

**Comment by ancorgs** [comment]
> @ancorgs
> 
> Maybe I should revert the commit related to the form and change the PR to improve only the "Add file system" button. What do you think?

Absolutely

**Comment by dgdavid** [comment]
> > @ancorgs
> > Maybe I should revert the commit related to the form and change the PR to improve only the "Add file system" button. What do you think?
> 
> Absolutely

Done, please review.

**Review by ancorgs — APPROVED** [review]
LGTM

**Pull request merged**